requirements.txt

```diff
@@ -1,14 +1,16 @@
-appdirs==1.4.0
-Django==1.10.5
-django-allauth==0.30.0
-django-rest-auth==0.9.0
-djangorestframework==3.5.3
-flake8==3.3.0
-mccabe==0.6.1
-packaging==16.8
-pycodestyle==2.3.1
-pyflakes==1.5.0
-psycopg2==2.6.2
-pyparsing==2.1.10
-six==1.10.0
-django-environ==0.4.3
+# Lint
+flake8==3.5.0
+
+# Django
+Django==2.0.1
```

Comment:
Should we really be using 2.0 as default? I've hit a few compat issues with 2.0

Reply:
Not sure yet, but they release as the last stable. I didn't see any major advantages other than simplifying a lot the URLs definition, but it's going to be a pain to upgrade if we need anything from 2.0. Do you remember any of these compatibility issues?

Reply:
I think you should embrace the future! Embrace Django 2.0!